src/app/services/webhooks.service.ts

```diff
@@ -158,33 +175,41 @@ const getConsoleMessage = (title, params) => {
 }
 
 // Formats webhook success info into an object to update Submissions collection
-const getSuccessDbUpdate = (response, { webhookUrl, signature }) => {
-  return { webhookUrl, signature, response: getFormattedResponse(response) }
+const getSuccessDbUpdate = (
+  response: AxiosResponse,
+  { webhookUrl, signature }: WebhookParams,
+): IWebhookResponse => {
+  return { webhookUrl, signature, ...getFormattedResponse(response) }
 }
 
 // Formats webhook failure info into an object to update Submissions collection
-const getFailureDbUpdate = (error, { webhookUrl, signature }) => {
-  const errorMessage = _.get(error, 'message')
-  const update = { webhookUrl, signature, errorMessage }
+const getFailureDbUpdate = (
+  error: Error | AxiosError,
+  { webhookUrl, signature }: WebhookParams,
+): IWebhookResponse => {
+  const errorMessage = get(error, 'message')
+  let update: IWebhookResponse = {
+    webhookUrl,
+    signature,
+    errorMessage,
+  }
   if (!(error instanceof WebhookValidationError)) {
-    update.response = getFormattedResponse(_.get(error, 'response'))
+    const { response } = getFormattedResponse(get(error, 'response'))
+    update.response = response
   }
   return update
 }
 
 // Formats a response object for update in the Submissions collection
-const getFormattedResponse = (response = {}) => {
+const getFormattedResponse = (
+  response: AxiosResponse,
```

Comment:
We lost the default assignment `= {}`, might result in undefined errors, since the callers of this function may not have been migrated to Typescript yet.

Reply:
I solve this by using lodash get operations within the function